app/features/app/components/App.js

```diff
@@ -35,17 +37,19 @@ export default class App extends Component<*> {
     render() {
         return (
             <AtlasKitThemeProvider mode = 'dark'>
-                <Router history = { history }>
-                    <Switch>
-                        <Route
-                            exact
-                            component = { Welcome }
-                            path = '/' />
-                        <Route
-                            component = { Conference }
-                            path = '/:domain?/:room' />
-                    </Switch>
-                </Router>
+                <Page navigation = { <Navbar /> }>
```

Comment:
We only want the navigation in the welcome page, so please move it there. While in a conference, we want it to take over the entire screen.

Reply:
I don't think we should do that.
Wouldn't user like to have access to drawers we will create in navigation while in the conference?

Reply:
They are not really useful IMHO. The conference itself has the ability to change things, that's why we discussed the syncing of our local settings with the ones the conference provides using the iframe API.

Maybe i'm used to seeing the conference component take the entire screen, so this looks weird to me.

@hristoterezov @gpolitis WDYT?

Reply:
@saghul 
Hmm. But then there will be different UI for settings on both components. 

PS - Syncing is not a problem. It will be easy.

Reply:
> Hmm. But then there will be different UI for settings on both components.

This will be the case no matter what, since the Meet UI does have the ability to set those.

Reply:
@saghul 
Yeah. 
Should I update this or wait for @hristoterezov and @gpolitis?

Reply:
I'd like them to weigh in and break the tie here, since you and I stand on opposite sides at the moment :-)

Reply:
I'm not a UI expert but I agree with Saul that jitsi-meet should provide all the necessary drawers/menus to configure the conference and also, in general, I prefer the interface that I'm working with to be as uncluttered as possible.

Reply:
@gpolitis Cool.
@saghul I have updated this.